dev-haskell/http-date/http-date-0.0.8.ebuild

```diff
@@ -9,15 +9,15 @@ CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
 inherit haskell-cabal
 
 DESCRIPTION="HTTP Date parser/formatter"
-HOMEPAGE="http://hackage.haskell.org/package/http-date"
+HOMEPAGE="https://hackage.haskell.org/package/http-date"
 SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
-RESTRICT=test # Ambiguous module name ‘Data.Time’: it was found in multiple packages: pulseaudio-0.0.2.1 time-1.8.0.2
+RESTRICT=test # Ambiguous module name âData.Timeâ: it was found in multiple packages: pulseaudio-0.0.2.1 time-1.8.0.2
```

Comment:
This change looks unintended.

Reply:
My editor did that for some reason :/
I fixed it now